### PR TITLE
add switch context to tree view title

### LINKED ISSUE #996: add action to switchContexts for the entire tree
Currently the switch user action will be changed to just switching between the `users` based on the cluster loggedin.

A new action will be added to the Explorer to switch between different contexts available in the kubeconfig.

### PR DESCRIPTION
**Pull request by mohitsuman**
fix #996

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -142,6 +142,15 @@
         "title": "Login into Cluster with credentials",
         "category": "OpenShift"
       },
+      {
+        "command": "openshift.explorer.switchContext",
+        "title": "Manage Contexts",
```
Comment (dgolovin) [inline package.json]: It is not really about managing contexts, but rather about changing contexts.

File: src/openshift/cluster.ts [hunk 2]
```diff
@@ -82,11 +82,11 @@ export class Cluster extends OpenShiftItem {
     static async switchContext() {
         const k8sConfig = new KubeConfigUtils();
         const contexts = k8sConfig.contexts.filter((item) => item.name !== k8sConfig.currentContext);
-        const contextName: QuickPickItem[] = contexts.map((ctx) => ({ label: `$(code)  ${ctx.name}`}));
-        const choice = await window.showQuickPick(contextName, {placeHolder: "Select the new context"});
+        const contextName: QuickPickItem[] = contexts.map((ctx) => ({ label: `${ctx.name}`}));
+        const choice = await window.showQuickPick(contextName, {placeHolder: "Select the new openshift context"});
```
Comment (dgolovin) [inline src/openshift/cluster.ts]: openshift => 'OpenShift'

File: package.json (hunk 1 above)
Comment (mohitsuman) [reply package.json]: There is a very funny thing here, the order in which the titles appear on the explorer bar is based on the title name. So to make the order correct I used something which starts after 'L' and before 'R' and hence Manage. Is there a way to sort the order ?

Comment (dgolovin) [reply package.json]: The same way  it is done in context menus I guess.

Comment (mohitsuman) [reply package.json]: No, it doesn't take group ordering. It just sorts name alphabetically.
And the contents displayed are only there if it's under `"group": "navigation"`.

Can we rename it to `Manage Context Switch` ?

Comment (mohitsuman) [reply package.json]: okay, i think I have a solution. sending the updates.

Comment (mohitsuman) [reply package.json]: Fixed by https://github.com/redhat-developer/vscode-openshift-tools/pull/1019/commits/886eb7f2b9b0df729efcf33a9e36466004005438

File: src/openshift/cluster.ts (hunk 2 above)
Comment (mohitsuman) [reply src/openshift/cluster.ts]: Fixed by https://github.com/redhat-developer/vscode-openshift-tools/pull/1019/commits/886eb7f2b9b0df729efcf33a9e36466004005438

**Comment by mohitsuman** [comment]
> ```
> error: server took too long to respond with version information.
> ```

Can you help in how you reproduced this issue, I can then have a look. 
Is this happening from context switch 3.x to 4.x or vice versa ?

**Comment by mohitsuman** [comment]
@dgolovin I have added a working scenario with the attached gif. This allows me to perform the context switching between different clusters. 
Here I am doing a switch between local 3.x and local 4.x cluster. The actions works seamlessly. Can you please check and let me know.

switch-ctxt

**Pull request merged**